Aztec/Classes/TextKit/TextView.swift

```diff
@@ -887,9 +887,13 @@ open class TextView: UITextView {
         let delay = 0.05
         DispatchQueue.main.asyncAfter(deadline: .now() + delay) {
             let pristine = self.selectedRange
-            let updated = NSMakeRange(max(pristine.location - 1, 0), 0)
+            var location = max(pristine.location - 1, 0)
+            if pristine.location == location {
+                location = min(pristine.location + 1, self.storage.length)
```

Comment:
Can we document this method, and anything that's not ovbious in it a bit more?

I'm realizing that reviewing this PR is difficult because it's not immediately obvious what the logic is.